Client/States/GameplayState.h

```diff
@@ -79,24 +78,20 @@ namespace NCL {
 
             Diagnostics packetsSent{};
 
-            
+            TextureBase* deathImageTex;
```

Comment:
Fine for now but we might end up having more UI elements on death and it could get a bit messy adding variables for all of them.

Reply:
Agreed! Pretty sure Joe adds elements to the canvas so I can look into that and refine the code.